plugin.json

```diff
@@ -47,7 +47,8 @@
                 "type": "bool",
                 "help_text": "When true, OAuth will be used as the authentication means with Zoom. \n When false, JWT will be used as the authentication means with Zoom. \n If you're currently using a JWT Zoom application and switch to OAuth, all users will need to connect their Zoom account using OAuth the next time they try to start a meeting. [More information](https://mattermost.gitbook.io/plugin-zoom/installation/zoom-configuration).",
                 "placeholder": "",
-                "default": false
+                "default": true,
```

Comment:
Just double checking this default value change is intended.

Reply:
@larkox Yes it's intended, since the OAuth feature is now our "main" feature offering, and we want to discourage using the JWT method.